HIP/hip-0000-cancun-3.md

```diff
@@ -0,0 +1,126 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Add Smart Contract Services Support for KZG Point Evaluation Precompiled Function
+author: Danno Ferrin (@shemnon)
+working-group: // TBD
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: // TBD
+discussions-to: // TBD
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+Add the Ethereum KZG Point Evaluation Precompile from EIP-4844: Shard Blob
+Transactions.
+
+## Motivation
+
+One part of [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844) is the KZG Point
+Evaluation function, a cryptographic precompile that can be used un zero
+knowledge proofs as well as proving that certain subsections of a blob are
+present without introducing the entire blob. This precompile adds value to the
+other parts of EIP-4844, but it does not require them. Hedera will expose
+support for this precompile without introducing other parts of the EIP.
+
+## Rationale
+
+While Hedera will not be supporting most EIP-4844 features the KZG Point
+Evaluation Precompile is cleanly severable from the other integrated features.
+This precompile, when accompanied by the associated trusted setup, can be viewed
+as a cryptographic primitive similar to ECDSA. The ability to proved subsets of
+hashed data provides unique functionality not found in other cryptographic
+precompiles. Ethereum apps may use the KZG features without any interation with
+blob varying transactions, and it makes sense to allow Hedera Smart Contract
+apps the same latitude.
+
+This precompile will be brought in with the same semantics and calling
+conventions as Ethereum Mainnet. The trusted setup from Ethereum Mainnet will
+also be used.
```

Comment:
> The trusted setup from Ethereum Mainnet will also be used.

What does this mean?

Reply:
Moon math.  It's a config file that people who are familiar with the algorithm will recognize.  Changed to `It also requires a customizable data file known as the trusted setup. For maximum compatibility the same trusted setup from Ethereum Mainnet will be used.`